approval_frame/templates/registration/password_reset_form.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <div class='row-fluid top-buffer'>
+      <div class='col-md-6 col-md-offset-3'>
+        <legend>Reset Password</legend>
+        <p>Enter your email address below and we will send you an email with
```

Comment:
Both legend and <p> can go inside the form. The instruction can be shortened to 
'Instructions to reset your password will be sent to the email address entered below.'